tests/EdFi.Admin.DataAccess.IntegrationTests/Repositories/AccessTokenClientRepoTests.cs

```diff
@@ -38,15 +41,23 @@ public class AccessTokenClientRepoTests : TestFixtureBase
         protected override void Arrange()
         {
             _transaction = new TransactionScope();
-
+          
             Factory = Stub<IUsersContextFactory>();
-
+           
+            var config = new ConfigurationBuilder()
+                .SetBasePath(Path.GetFullPath(Path.Combine(Environment.CurrentDirectory, @"..\..\.."))+"\\")
```

Comment:
We should be using TestContext.CurrentContext.TestDirectory for the base path, also remove the relative paths. set the appsettings.json to copy always